ldk/javascript/examples/self-test-loop/src/tests/index.ts

```diff
@@ -814,28 +827,108 @@ export const networkHTTP = (): Promise<boolean> =>
       });
   });
 
+const finaliseWebsocketTest = async (cancellable: Cancellable, socket: network.Socket) => {
```

Comment:
"finalizeWebsocketTest"

Reply:
This is a good candidate for a `utilities` file or something. I get why you added it here but it feels a bit out of place.

Reply:
updated